src/main/resources/api.yaml

```diff
@@ -235,3 +298,21 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+
```

Comment:
Jak zacząłeś inicjatywę newline'ową, to zapraszam do wcielenia jej bardziej globalnie